# MATH 2441 Term Test C Solutions (Version 2)

#### Question 1

A wine importer needs to report the average percentage of alcohol in bottles of Chilean wine. From experience with previous kinds of wine, the importer believes the population standard deviation is 0.8\%. The importer randomly samples 50 bottles of the new wine and obtains a sample mean $\bar{x}=$8.6\%. Give a 90\% confidence interval for the average percentage of alcohol in all bottles of the new wine.

<img src="./diagrams/hmdyspyd.png" height=33% width=33%>

In [8]:
zat<-qnorm(.95);print(zat)

[1] 1.644854


#### Question 2

The New England Journal of Medicine published preliminary results of an ongoing trial with the antiviral drug remdesivir on April 10, 2020. Even though the sampling is imperfect, at the current state of research here is the best we can do: 7 out of 53 hospitalized patients with SARS-CoV-2 infections died if treated with remdesivir (in North America, Europe, and Japan), compared to a group of 201 hospitalized cases in Wuhan, China, 44 of which died. Assume the requirements for a hypothesis test are met and test the claim that remdesivir is more effective than non-treatment at a significance level of 0.1 (given the urgency of the situation, we cannot be picky with a high significance level). Use the $p$-value approach.

<img src="./diagrams/ujuzaxaj.png" height=33% width=33%>

In [1]:
z<-(((7/53)-(44/201))/(((((51/254)*(203/254))/53)+((((51/254)*(203/254))/201)))^.5));print(z)

[1] -1.403749


In [2]:
pnorm(z)

[1] 0.08019677

#### Question 3

Researchers measured skulls from different time periods in an attempt to determine whether interbreeding of cultures occurred. Use a 0.01 significance level to test the claim that the mean maximal skull breadth in 4000 BCE is less than the mean in 150 AD.

In [13]:
n1=30;x1=131.37;s1=5.13;n2=30;x2=136.17;s2=5.35

In [15]:
t=(x1-x2)/(((s1^2/n1)+(s2^2/n2))^.5);print(t)

[1] -3.54699


In [19]:
tc<-qt(0.01,29);print(tc)

[1] -2.462021


Reject $H_{0}$.

#### Question 4

The average weekly earnings for all full-time-equivalent employees are reported to be \\$351. Suppose that you want to check this claim since you believe it is too low. You want to prove that average weekly earnings of all employees are higher than the amount stated. You collect a random sample of 800 employees in all areas and find that the sample mean is \\$360 and the sample standard deviation is \\$94. Can you disprove the claim? (Use a 0.05 significance level.)

$\mu=351,n=800,\bar{x}=360,s=94,1-\alpha=0.95$

$$t^{\ast}=\frac{\bar{x}-\mu}{\frac{s}{\sqrt{n}}}$$

In [14]:
ts<-(360-351)/(94/(800^.5))

In [15]:
ts

2.70806852369316

$$t_{\alpha}=1.646$$

In [6]:
talpha<-qt(.95,799);print(talpha)

[1] 1.646763


Decision: reject $H_{0}$.

#### Question 5

$\sigma^{2}=8100,n=36,s^{2}=11256,1-\alpha=0.95$

$$\chi^{2}=\frac{(n-1)s^{2}}{\sigma^{2}}=\frac{35\cdot{}11256}{8100}$$

In [8]:
cs<-(35*11256)/8100

In [9]:
cs

48.637037037037

In [10]:
crit<-c((16.791+24.433)/2,(46.979+59.342)/2)

In [11]:
crit

20.612 
 53.1605

Decision: fail to reject $H_{0}$.

<h4>Archive</h4>

#### Question 2

Do people who work for non-profit organizations differ from those who work at for-profit companies when it comes to personal job satisfaction? Separate random samples were collected by a polling agency to investigate the difference. Data collected from 422 employees at non-profit organizations revealed that 377 of them were highly satisfied. From the for-profit companies, 441 out of 518 employees reported the same level of satisfaction. The researchers want to test if the proportions of highly satisfied employees are the same at for-profit companies as at non-profit organizations. What do you conclude at a confidence level of 0.95?

In [1]:
prop.test(c(377,441),c(422,518),alternative="two.sided")


	2-sample test for equality of proportions with continuity correction

data:  c(377, 441) out of c(422, 518)
X-squared = 3.272, df = 1, p-value = 0.07047
alternative hypothesis: two.sided
95 percent confidence interval:
 -0.002629931  0.086657086
sample estimates:
   prop 1    prop 2 
0.8933649 0.8513514 


In [8]:
n1=441;n2=518;x1=377;x2=441;p1=x1/n1;p2=x2/n2

In [9]:
pb=(p1+p2)/(n1+n2);qb=1-pb

In [10]:
z=(p1-p2)/(((pb*qb/n1)+(pb*qb/n2))^.5);print(z)

[1] 1.290565


In [12]:
zc<-qnorm(0.975);print(zc)

[1] 1.959964


#### Question 5

$\sigma=0.008,n=50,\bar{x}=0.086,1-\alpha=0.90$

$$E=z_{\frac{\alpha}{2}}\frac{\sigma}{\sqrt{n}}$$

In [16]:
E<-1.645*(0.008/(50^.5))

In [17]:
E

0.00186110504808299

In [18]:
confint<-c(0.086-E,0.086+E)

In [19]:
confint

0.084138894951917 
 0.087861105048083

The confidence interval is between 8.41% and 8.79%.

#### Question 1

$n=502,x=451,1-\alpha=0.95$

$$E=z_{\frac{\alpha}{2}}\sqrt{\frac{\hat{p}\hat{q}}{n}}=1.96\sqrt{\frac{\frac{451}{502}\frac{51}{502}}{502}}$$

In [1]:
E<-1.96*(((451/502)*(51/502))/502)^.5

In [2]:
E

0.0264285815448348

In [3]:
lr<-(451/502)-E

In [4]:
hr<-(451/502)+E

In [5]:
interval<-c(lr,hr)

In [7]:
interval

0.871977792957157 
 0.924834956046827

#### Question 3

$p^{\ast}=0.18,E=0.04,1-\alpha=0.99$

$$n=\frac{(z_{\frac{\alpha}{2}})^{2}p^{\ast}(1-p^{\ast})}{E^{2}}$$

In [12]:
n<-(2.575^2*0.18*(1-0.18))/(0.04^2)

In [13]:
n

611.67515625